src/tests_e2e/e2e_test.py

```diff
@@ -40,7 +40,12 @@ def test_app():
 
 @pytest.fixture(scope="module")
 def driver():
-    with webdriver.Chrome() as driver:
+    opts = webdriver.ChromeOptions()
+    opts.add_argument('--headless')
+    opts.add_argument('--no-sandbox')
+    opts.add_argument('--disable-dev-shm-usage')
+    with webdriver.Chrome('./chromedriver', options=opts) as driver:
```

Comment:
Can't you keep the location of the chromedriver flexible? `webdriver.Chrome(options=opts)`?